content/packit-as-a-service.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Packit-as-a-Service"
```

Comment:
can we name it Documentation?

Reply:
It makes sense to me if `packit.dev` is web only about `Packit-as-a-Service`. But If we talk about `Packit` and `Packit-as-a-Service`, then I wanted to differentiate it a bit. That was my proposal.

Reply:
Yes, that's a good point. In that case, packit-as-a-service makes sense.

On the flip side, I'd say that our end-goal is packit-as-a-service where packit tool is really just a detail of the complete solution.

Oki, let's leave it as it is.